Sources/App/Controllers/API/Package+Badge.swift

```diff
@@ -67,15 +69,18 @@ extension Package {
                 label = "Swift Compatibility"
         }
 
+        let message = badgeMessage(badgeType: badgeType)
         return Badge(schemaVersion: 1,
                      label: label,
-                     message: badgeMessage(badgeType: badgeType),
-                     color: "blue",
-                     cacheSeconds: cacheSeconds)
+                     message: message ?? "unavailable",
+                     isError: message == nil,
+                     color: message == nil ? "inactive" : "F05138",
```

Comment:
Shows the brand colour only if it's all working fine. If we're in "unavailable" mode for whatever reason, it's set to the inactive grey colour.

isError purely means that the colour can't be overridden which means it's always the inactive grey colour when we can't load the details. They can override the brand colour should they choose to do so.

This logic wasn't on the original issue but I hope you agree it makes sense given the context!